cli/src/prometheus.rs

```diff
@@ -252,6 +252,41 @@ pub fn write_solido_metrics_as_prometheus<W: io::Write>(
             metrics: solido_histogram_to_metrics(at, &metrics.deposit_amount),
         },
     )?;
+
+    write_metric(
+        out,
+        &MetricFamily {
+            name: "solido_withdraw_count_total",
+            help: "Total number of withdrawals done by users.",
```

Comment:
Nit: "Total number of withdrawals done by users." -> "Total number of withdrawals made by users."

Reply:
Fixed.